packages/dynamite/dynamite_end_to_end_test/lib/parameters.openapi.json

```
@@ -31,6 +31,57 @@
                                 }
                             }
                         }
+                    },
+                    {
+                        "name": "array",
+                        "in": "query",
+                        "schema": {
+                            "type": "array"
+                        }
+                    },
+                    {
+                        "name": "bool",
+                        "in": "query",
+                        "schema": {
+                            "type": "boolean"
+                        }
+                    },
+                    {
+                        "name": "string",
+                        "in": "query",
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    {
+                        "name": "string_binary",
+                        "in": "query",
+                        "schema": {
+                            "type": "string",
+                            "format": "binary"
+                        }
+                    },
+                    {
+                        "name": "integer",
```

```suggestion
                        "name": "int",
```
To properly check for collision with `int`